ui/webui/src/components/installation/ReviewConfiguration.jsx

```diff
@@ -14,16 +14,143 @@
  * You should have received a copy of the GNU Lesser General Public License
  * along with This program; If not, see <http://www.gnu.org/licenses/>.
  */
-import React from "react";
+import cockpit from "cockpit";
+import React, { useEffect, useState } from "react";
 
 import {
-    Form
+    Checkbox, Title,
+    DataList, DataListItem,
+    DataListItemRow, DataListItemCells,
+    DataListCell,
+    DescriptionList, DescriptionListGroup,
+    DescriptionListTerm, DescriptionListDescription,
+    Flex, Popover,
 } from "@patternfly/react-core";
 
+import {
+    getSelectedDisks, getDeviceData,
+} from "../../apis/storage.js";
+
+import {
+    getLanguage, getLanguageData,
+} from "../../apis/localization.js";
+
+import HelpIcon from "@patternfly/react-icons/dist/esm/icons/help-icon";
+
+const _ = cockpit.gettext;
+
 export const ReviewConfiguration = () => {
+    const [systemLanguage, setSystemLanguage] = useState("YYY");
+    const [selectedDisks, setSelectedDisks] = useState();
+    const [deviceData, setDeviceData] = useState({});
+
+    useEffect(() => {
+        getLanguage()
+                .then(res => {
+                    console.info("language code");
+                    console.info({ res });
+                    getLanguageData({ lang: res }).then(res => {
+                        console.info("language data");
+                        console.info({ res });
+                        setSystemLanguage(res["native-name"].v);
+                    }, console.error);
+                }, console.error);
+    }, []);
+
+    useEffect(() => {
+        getSelectedDisks()
+                .then(res => {
+                    console.info("selected disks");
+                    console.info({ res });
+                    setSelectedDisks(res);
+                    // get detailed data for the selected disks
+                    res.forEach(disk => {
+                        console.info("processing disk");
+                        console.info(disk);
+                        getDeviceData({ disk })
+                                .then(res => {
+                                    console.info("disk device data");
+                                    console.info(res);
+                                    setDeviceData(d => ({ ...d, [disk]: res[0] }));
+                                }, console.error);
+                    });
+                }, console.error);
+    }, []);
+
+    // handle case of disks not (yet) loaded
+    if (!selectedDisks) {
+        return null;
+    }
+
+    return (
+        <>
```

Comment:
When you remove the TItle you can also remove the <> wrapper.

Reply:
Removed. :)

What is it doing actually ? Just a generic parent component in case there is not any specific one ?

Reply:
Answered the what is it doing part in a comment above.